AK/Optional.h

```diff
@@ -162,7 +162,7 @@ class [[nodiscard]] Optional {
     ALWAYS_INLINE T* operator->() { return &value(); }
 
 private:
-    alignas(T) u8 m_storage[sizeof(T)];
+    AK::aligned_storage<sizeof(T), alignof(T)>::type m_storage;
```

Comment:
Would it make sense to create a template overload of your new `AK::AlignedStorage` that takes a T and then you can do the `sizeof(T)` and `alignof(T)` internal to the template, so you don't have to do that in the common case?